docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Add cron entry
+echo "$CRON_SCHEDULE /usr/local/bin/npm start --prefix /app/" | crontab -
```

Comment:
Since this is in the entrypoint, won't this get re-added to crontab every time the container starts?
Fine if you create a new container each time, but for restarted ones I think you'd end up with running multiple instances of npm script on the schedule.

Reply:
Good thought, but it's actually ok. The way the crontab command works in Alpine just replaces the root crontab with whatever is supplied. So it replaces it each restart, which ensures that if someone changes the ENV value it will get replaced on container restart.